alvr/filesystem/src/lib.rs

```diff
@@ -335,5 +346,5 @@ pub fn filesystem_layout_from_openvr_driver_root_dir(dir: &Path) -> Layout {
 pub fn filesystem_layout_invalid() -> Layout {
     LAYOUT_FROM_ENV
         .clone()
-        .unwrap_or_else(|| Layout::new(Path::new("")))
+        .unwrap_or_else(|| Layout::new(Path::new("./")).unwrap())
```

Comment:
Did you ensure this works as intended or would this not present a change in behavior?

Reply:
Lets merge and see if we get bug reports